io.catenax.user_estimated_loading/1.0.0/UserEstimatedLoading.ttl

```diff
@@ -0,0 +1,244 @@
+   samm:description "characteristic describing the basic technical data of the vehicle "@en ;
+   samm:dataType :VehicleDataEntity .
+
+:HistoryProfileList a samm-c:List ;
+   samm:preferredName "history profile list"@en ;
+   samm:description "characteristic of historie profile list\n"@en ;
+   samm:dataType :HistoryProfilesEnity .
+
+:FutureList a samm-c:List ;
+   samm:preferredName "future usage list"@en ;
+   samm:description "characteristic of future usages"@en ;
+   samm:dataType :FutureListEntity .
+
+:VehicleStatusEntity a samm:Entity ;
+   samm:preferredName "vehicle status entity"@en ;
+   samm:description "vehicle status is basically described by mileage and age"@en ;
+   samm:properties ( [ samm:property :mileage; samm:optional true ] [ samm:property :ageInMonth; samm:optional true ] ) .
+
+:VehicleDataEntity a samm:Entity ;
+   samm:preferredName "vehicle data entity"@en ;
+   samm:description "basic technical data of the vehicle as specified in KBA data"@en ;
+   samm:properties ( [ samm:property :vehicleType; samm:optional true ] [ samm:property :numberOfDriveAxle; samm:optional true ] [ samm:property :energySourceCode; samm:optional true ] :powerKW :grossVehicleWeightKG ) .
+
+:HistoryProfilesEnity a samm:Entity ;
+   samm:preferredName "History profiles entity"@en ;
+   samm:description "collection of previous usage profiles "@en ;
+   samm:properties ( :percentage [ samm:property :trailerUse; samm:optional true ] [ samm:property :cityDriving; samm:optional true ] :drivingStyle :hilliness :curviness ) .
+
+:FutureListEntity a samm:Entity ;
+   samm:preferredName "future list entity"@en ;
+   samm:description "List of future scenarios"@en ;
+   samm:properties ( :futureScenario :futureScenarioId ) .
+
+:mileage a samm:Property ;
+   samm:preferredName "mileage @en"@en ;
+   samm:description "distance the vehicle has travelled in kilometre"@en ;
+   samm:characteristic :Mileage ;
+   samm:exampleValue "123456"^^xsd:positiveInteger .
+
+:ageInMonth a samm:Property ;
+   samm:preferredName "vehicle age @en"@en ;
+   samm:description "age of the vehicle in month"@en ;
+   samm:characteristic :AgeInMonths ;
+   samm:exampleValue "42"^^xsd:nonNegativeInteger .
+
+:vehicleType a samm:Property ;
+   samm:preferredName "vehicle variant"@en ;
+   samm:description "Vehicle variant - body shapes like SUV, Van, Limousine"@en ;
+   samm:characteristic :KbaVariant ;
+   samm:exampleValue "SUV" .
+
+:numberOfDriveAxle a samm:Property ;
+   samm:preferredName "number of drive axles"@en ;
+   samm:description "identification of 4WD, default 1: 2WD"@en ;
+   samm:characteristic :NumberOfDriveAxle ;
+   samm:exampleValue "2"^^xsd:positiveInteger .
+
+:energySourceCode a samm:Property ;
+   samm:preferredName "energy source  code"@en ;
+   samm:description "the code for the energy source respectively fuel type of the vehicle according to KBA list"@en ;
+   samm:characteristic :EnergySource ;
+   samm:exampleValue "\"0002\"" .

```

Comment:
I think the escaped quotation marks are not required as you have plain strings in the enumeration. Possible to remove \"?